modules/ingestor/src/graph/sbom/clearly_defined.rs

```diff
@@ -1,55 +1,62 @@
 use crate::graph::purl::creator::PurlCreator;
 use crate::graph::sbom::{LicenseCreator, LicenseInfo, SbomContext, SbomInformation};
-use sea_orm::ConnectionTrait;
+use sea_orm::{ConnectionTrait, EntityTrait, Set};
+use sea_query::OnConflict;
 use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
 use tracing::instrument;
 use trustify_common::purl::Purl;
+use trustify_entity::sbom_package_license;
+use uuid::Uuid;
 
 impl SbomContext {
     #[instrument(skip(db, curation), err)]
     pub async fn ingest_clearly_defined_curation<C: ConnectionTrait>(
         &self,
         curation: Curation,
         db: &C,
+        sbom_id: Uuid,
+        node_id: String,
```

Comment:
To my understanding clearly defined doesn't have a real "node ID", right? If that's the case, then we could use a static/magic/const value instead. Like we do it with the CyclondeDX root node. In the context of the "SBOM", that would always be unique.

Reply:
It was based on this code that I started using the document ID as the node ID.
https://github.com/trustification/trustify/blob/0224bef3af458bca3be3ce30763915dd0d6789f3/modules/ingestor/src/graph/sbom/clearly_defined.rs#L92

Reply:
I believe it is also acceptable, but it would be better to use the same method in the same place.

Reply:
What's the benefit?

Reply:
It is strange to use two different node id definitions in the same clear-defined .

Reply:
https://github.com/trustification/trustify/blob/0224bef3af458bca3be3ce30763915dd0d6789f3/modules/ingestor/src/graph/sbom/clearly_defined.rs#L92

Reply:
At this point, the document incorporates the content of the coordinates from the curation, such as "crate/cratesio/-/chrono." In my view, this approach is more meaningful in practice.

Reply:
The "node id" is an unique ID in the context of an SBOM. Having a static node id makes it predictable. As the value itself is not important, it wouldn't matter.

Unless there are reasons against it, I'd like to use a static value.

Reply:
What static value should we use? “ClearlyDefinedCuration”.

Reply:
Sounds reasonable.